cbmc/proofs/IotMqtt_SubscribeAsync/IotMqtt_SubscribeAsync_harness.c

```diff
@@ -0,0 +1,107 @@
+#include "iot_config.h"
+#include "private/iot_mqtt_internal.h"
+
+#include <stdlib.h>
+
+#include "mqtt_state.h"
+
+/**
+ * We abstract all functions related to concurrency and assume they are correct.
+ * Thus, we add these stub to increase coverage.
```

Comment:
Say instead "We constrain the return values of these functions because they are checked by assertions in the MQTT code."  This is not about coverage.

Reply:
Done!